extensions-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/ProcessAction.scala

```diff
@@ -29,7 +31,8 @@ object ProcessActionType extends Enumeration {
   val UnArchive: Value = Value("UNARCHIVE")
   val Pause: Value = Value("PAUSE") //TODO: To implement in future..
 
-  val defaultActions: List[ProcessActionType] = Nil
+  val DefaultActions: List[ProcessActionType] = Nil
+  val StateActions = List(Cancel, Deploy)
```

Comment:
I would inline this in the place of usage so it will be more clear why this set contains only cancel and deploy and also add a comment next to it.

Reply:
After rethinking, we have to move this from `extensions-api` to `designer`. Now I see that it is in an API module. It can't be here:
- Nobody writing and extensions suppose to use it, we don't have such a use case
- Each change in an API module should be described in MigrationGuide. We don't want to do this.
Please move it to `DBFetchingProcessRepository` where it is only used.
 
BTW change above should be described in MigrationGuide as well.

Reply:
I disagree, when I move `StateActions` then we don't change API, so there is no need to change MigrationGuide? But I've added some info about the change.